test_requirements.txt

```diff
@@ -1,3 +1,4 @@
+botocore
 py==1.5.1
```

Comment:
Is this a known botocore issue, or is it one of our other packages?

Reply:
I don't think its even an "issue". They said there python-dateutil requirement is <2.7.0 and then apparently python-dateutil had a promotion to 2.7 so now that's an issue if anything else is installed before it decides to take latest